net.shadps4.shadPS4.yaml

```diff
@@ -3,6 +3,7 @@ runtime: org.kde.Platform
 runtime-version: '6.8'
 sdk: org.kde.Sdk
 command: shadps4
+desktop-file-name-suffix: ' Nightly'
```

Comment:
We need to discuss what name to use for nightly version here.

Reply:
a nightly suffix is fine with me.  since theres no clear beta delineation, nightly is more accurate imo as long as we update it every day

Reply:
hmm, so we have to change the ci to update everyday then

Reply:
@georgemoralis Is the name ``shadPS4 Nightly`` for nightly version sound good to you?